frontend/src/components/studentComponent.tsx

```diff
@@ -1,15 +1,26 @@
 import React from "react";
 import "./studentComponent.css";
+import { removeStudent } from "../services/LoginService";
 
 type StudentComponentProps = {
     studentName: string;
 };
 
 const StudentComponent: React.FC<StudentComponentProps> = ({ studentName }) => {
+    const handleRemoveStudent = (name: string) => {
+        removeStudent(name)
```

Comment:
Tak jak Dominik Puz już wspominał, imię studenta powinno zawierać "%20" zmiast " ", bo ścieżka nie może zawierać spacji. Axios dodaje sam na miejsce spacji '+', nie jestem pewny czy java po przyjęciu takiej ścieżki z powrotem by zamienił '+' na ' ' (z tego co wstępnie patrzyłem to nie)

najprościej to naprawić poprzez
`name.replace(' ', "%20");`